docs/quick-start.md

```diff
@@ -72,18 +72,17 @@ Once Numaflow is installed, create a simple Numalogic pipeline, which takes in t
 
 For building this pipeline, navigate to [numalogic-simple-pipeline](https://github.com/numaproj/numalogic/tree/main/examples/numalogic-simple-pipeline) under the examples folder and execute the following commands.
 
-1. Build the docker image, import it to k3d, and apply the pipeline.
+1. Apply the pipeline. *Note Make sure the pipeline and, numaflow controllers and isbsvc pods are running in the same namespace (`default` in this case).*
```

Comment:
@ab93, the default installation will install the numaflow controller in `numaflow-system`. are we providing a different installation strategy? /cc @whynowy

Reply:
@vigith we changed this from `numaflow-system`.

Reply:
> 

@ab93 - Where is it mentioned?